### PR TITLE
[vSphere][vSAN] Fixing some review comments and using snake case for measurement names

### PR DESCRIPTION
**Pull request by ambekarnikhil**
Merge request to fix reviews for https://github.com/influxdata/telegraf/pull/5970

### CODE REVIEW

File: plugins/inputs/vsphere/vsan.go [hunk 1]
```diff
@@ -371,29 +382,32 @@ func (e *Endpoint) queryResyncSummary(ctx context.Context, vsanClient *soap.Clie
 		return nil
 	}
 	hostRefValue := hosts[0].Reference().Value
-	vsanSystemEx := types.ManagedObjectReference{
-		Type:  "VsanSystemEx",
-		Value: fmt.Sprintf("vsanSystemEx-%s", strings.Split(hostRefValue, "-")[1]),
-	}
+	hostRefValueParts := strings.Split(hostRefValue, "-")
+	if len(hostRefValueParts) == 2 {
```
Comment (YuxinZhou) [inline plugins/inputs/vsphere/vsan.go]: I feel it is easier to read if you write like, 
```
if len(hostRefValueParts) != 2 {
    return err
}
```

File: plugins/inputs/vsphere/README-VSAN.md [hunk 2]
```diff
@@ -211,10 +211,10 @@ For performance consideration, it is better to specify two instances of the plug
...
+vsphere_vsan_performance_hostdomclient,clustername=Example-VSAN,dcname=Example-DC,host=host.example.com,hostname=DC0_C0_H0,moid=domain-c8,source=Example-VSAN,vcenter=localhost:8898 iops_read=7,write_congestion=0,unmap_congestion=0,read_count=2199,iops=8,latency_max_write=8964,latency_avg_unmap=0,latency_avg_write=1883,write_count=364,num_oio=12623,throughput=564127,client_cache_hits=0,latency_max_read=17821,latency_max_unmap=0,read_congestion=0,latency_avg=1154,congestion=0,throughput_read=554721,latency_avg_read=1033,throughput_write=9406,client_cache_hit_rate=0,iops_unmap=0,throughput_unmap=0,latency_stddev=1315,io_count=2563,oio=4,iops_write=1,unmap_count=0 1578955200000000000
+vsphere_vsan_performance_clusterdomcompmgr,clustername=Example-VSAN,dcname=Example-DC,host=host.example.com,moid=domain-c7,source=Example-VSAN,uuid=XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXX,vcenter=localhost:8898 latency_avg_rec_write=0,latency_avg_write=9886,congestion=0,iops_resync_read=0,lat_avg_resync_read=0,iops_read=289,latency_avg_read=1184,throughput_write=50137368,iops_rec_write=0,throughput_rec_write=0,tput_resync_read=0,throughput_read=9043654,iops_write=1272,oio=97 1578954900000000000
```
Comment (YuxinZhou) [inline plugins/inputs/vsphere/README-VSAN.md]: so on the Wavefront, the metrics name will be `vsphere.vsan.performance.clusterdomcompmgr. latency.avg.rec.write`, right?

File: plugins/inputs/vsphere/vsan.go (hunk 1 above)
Comment (ambekarnikhil) [reply plugins/inputs/vsphere/vsan.go]: I have changed the codes

File: plugins/inputs/vsphere/README-VSAN.md (hunk 2 above)
Comment (ambekarnikhil) [reply plugins/inputs/vsphere/README-VSAN.md]: Yes. That's correct. Thanks for the review @YuxinZhou

**Pull request merged**